website/content/gateway/api_reference/resources/authorizations/authorizations.md

```diff
@@ -85,6 +85,10 @@ If the latest approved authorization in the series was not processed via Clearha
 Conditional. Cannot be present if `series[type]` is present.
 {{% /description_details %}}
 
+{{% description_term %}}sca_exemption[transaction_risk_analysis] {{% regex %}}(true|false){{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Whether or not to request Transaction Risk Analysis (TRA) SCA exemption.
+{{% /description_details %}}
+
```

Comment:
Is this the right place to place it? If we won't end up supporting many different sca exemption types I think so, but if we already know that we will end up with 3 or more I think we should create a section dedicated to sca exemptions.

Reply:
I lean towards making such "refactoring" when it is needed and not now.

I chose this place because the parameters appear to be in alphabetic order.